lib/gooddata/lcm/actions/collect_meta.rb

```diff
@@ -14,7 +14,7 @@ class CollectMeta < BaseAction
 
       PARAMS = define_params(self) do
         description 'Production Tag Name'
-        param :production_tag, instance_of(Type::StringType), required: false
+        param :production_tags, instance_of(Type::StringType), required: false
```

Comment:
We should maintain backward compatibility (i.e. support both `production_tag` and `production_tags`).